account_cash_discount_base/README.rst

```diff
@@ -0,0 +1,80 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.png
+    :alt: License: AGPL-3
+
+==========================
+Account Cash Discount Base
+==========================
+
+This module was written to allow you to define cash discount (aka early payment discount)
+on customer or supplier invoices.
```

Comment:
@benwillig @sbidoul is this similar to https://github.com/OCA/account-payment/pull/184 ?

Reply:
Hi @rafaelbn ! This is the same domain, for sure. #184 seems to be about configuring the payment term only. Here we make it easy to pay supplier invoices with early payment discount and account_payment_order (including automatic reconciliation, proper handling of taxes with configurable rules etc). So there is little overlap it seems. At some point we considered using payment terms, which we rejected (I don't remember all the details now but one of the reason is that it was much more complex to handle)?